packages/core/src/components/document-component-demo/readme.md

```diff
@@ -7,18 +7,21 @@
 
 ## Properties
 
-| Property      | Attribute      | Description                                      | Type                  | Default     |
-| ------------- | -------------- | ------------------------------------------------ | --------------------- | ----------- |
-| `jsonFile`    | `json-file`    | Name of the file to load.                        | `string \| undefined` | `undefined` |
-| `openDefault` | `open-default` | The default state for all Document Components.   | `boolean`             | `false`     |
-| `showCanvas`  | `show-canvas`  | Show canvas to where Document Component extends. | `boolean`             | `false`     |
+| Property      | Attribute      | Description                                                                                          | Type                                | Default      |
+| ------------- | -------------- | ---------------------------------------------------------------------------------------------------- | ----------------------------------- | ------------ |
+| `href`        | `href`         | The URL to which the Heading links (only in mode="table-of-contents").                               | `string \| undefined`               | `undefined`  |
```

Comment:
Ik zou `href` geen property maken die vanuit Storybook kan worden gezet, maar deze genereren voor mode Table of Contents

Reply:
Doe nu dit

```ts
href: a.mode === "table-of-contents" ? "/document/id" : undefined,
```

in de `documentComponentMapper` function